src/ngraph/frontend/onnx_import/ops_bridge.cpp

```diff
@@ -337,6 +338,7 @@ namespace ngraph
             REGISTER_OPERATOR("Relu", 1, relu);
             REGISTER_OPERATOR("Reshape", 1, reshape);
             REGISTER_OPERATOR("ReverseSequence", 1, reverse_sequence);
+            REGISTER_OPERATOR("ROIAlign", 1, roi_align);
```

Comment:
Name of this operator in ONNX has lowercase "oi" in Roi
```suggestion
            REGISTER_OPERATOR("RoiAlign", 1, roi_align);
```